.github/workflows/trivy-ubuntu16.yml

```diff
@@ -0,0 +1,41 @@
+name: Trivy Analysis (indy_node:ubuntu16)
+
+on:
+  schedule:
+    # run every Wednesday at 23:20
+    - cron: '20 23 * * 3'
+  workflow_dispatch:
+
+jobs:
+  trivy_analysis:
+    name: Trivy Analysis (indy_node:ubuntu16)
+    runs-on: "ubuntu-18.04"
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Run Trivy on indy_node:ubuntu16
+        uses: aquasecurity/trivy-action@master
+        with:
+          image-ref: 'ghcr.io/idunion/indy-node-container/indy_node:ubuntu'
```

Comment:
ubuntu16

Reply:
done and for `/trivy-ubuntu19.yml` as well